api/config/config_stable.json

```diff
@@ -30,5 +30,8 @@
     "adminIdVerification": "stable-environment-manual-id-verification-requests@fake-research-aou.org",
     "supportGroup": "all-of-us-workbench-eng@googlegroups.com",
     "verifiedSendingAddress": "all-of-us-workbench-eng@googlegroups.com"
+  },
+  "mandrill": {
+    "fromEmail": "donotreply@fake-research-aou.org"
```

Comment:
I think we have been using `stable.fake-research-aou.org` for stable (and similarly for staging).